examples/enum_ostream.cpp

```diff
@@ -1,4 +1,4 @@
-#include <CLI/CLI.hpp>
+#include "CLI/CLI.hpp"
```

Comment:
Why is change? This is not a relative import, so it should be `<>`, IMO. The main header files use `""`, but that's a special case, since the single file header script uses it to detect local includes.

Reply:
My main goal here was consistency in the examples. The majority used `""`, so I went with that. But you're right, let me change them all to  `<>`.

Reply:
Actually, the changeset is already large enough in this PR. Let me make the includes in the examples consistent in another PR.